app/javascript/modules/listingDetailsLottery/ListingDetailsLotteryPreferences.tsx

```diff
@@ -0,0 +1,86 @@
+import React from "react"
+import { t } from "@bloom-housing/ui-components"
+import dayjs from "dayjs"
+import { RailsLotteryBucketsDetails } from "../../api/types/rails/listings/RailsLotteryBucketsDetails"
+import { RailsListing } from "../listings/SharedHelpers"
+
+export interface ListingDetailsLotteryPreferencesProps {
+  listing: RailsListing
+  lotteryBucketsDetails: RailsLotteryBucketsDetails
+}
+
+export const ListingDetailsLotteryPreferences = ({
+  listing,
+  lotteryBucketsDetails,
+}: ListingDetailsLotteryPreferencesProps) => {
+  return (
+    <div className="text-tiny">
+      <p className="py-4 text-sm">{t("lottery.bucketsIntro")}</p>
+      <h2 className="text-caps-underline">{t("lottery.housingPreferences")}</h2>
```

Comment:
We can likely using the Heading component here, so the styles stay consistent, for all headers with the `text-caps-underline` class
https://storybook.bloom.exygy.dev/?path=/story/headers-heading--sidebar-header